### ISSUE TITLE
`InvalidStateError` in Firefox when using `sound` component

### ISSUE THREAD

**Issue by dsinni** [aframevr/aframe#3845]
**Description:**

I'm receiving `InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable` nearly consistently when using the `sound` component in Firefox with A-Frame `master`.

- A-Frame Version: A-Frame `master`, Commit `#6fde084`
- Platform / Device: Windows 10 / Firefox
- Reproducible Code Snippet or URL:

**Code:** https://glitch.com/edit/#!/neon-kingfisher

**Demo:** https://neon-kingfisher.glitch.me

I'm using the same demo to reproduce and demonstrate #3844

**Comment by dsinni** [comment]
Yeah, possibly. However, the `a-assets` `timeout` is 30 seconds and the audio has `preload="auto"`, yet the scene loads nearly immediately in most cases.

Shouldn't that block the scene load until the either the asset loads or timeout?

**Comment by dmarcos** [comment]
@ngokevin Anything to do here?
